app/pandas.ipynb

```diff
@@ -271,24 +696,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
    "source": [
     "# Data\n",
     "data1 = {'Name': ['John', 'Emily', 'Ryan'],\n",
     "         'Age': [25, 28, 22]}\n",
     "data2 = {'Name': ['Emily', 'Ryan', 'Mike'],\n",
     "         'City': ['Los Angeles', 'Chicago', 'Houston']}\n",
     "\n",
-    "# create DataFrame objects"
+    "df1 = pd.DataFrame(data1)\n",
+    "df2 = pd.DataFrame(data2)\n",
+    "df_concat = pd.concat([df1, df2])\n",
+    "df_merged = pd.merge(df1, df2, on=\"Name\", how=\"outer\")\n",
+    "print(df_concat, \"\\n\", df_merged)"
```

Comment:
The concatenation of `df1` and `df2` using `pd.concat` will result in a DataFrame with NaN values for missing columns. Consider specifying the axis or handling NaN values if needed.